lib/webhooks/index.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpStatusCode } from 'axios';
+import express, { Request, Response } from 'express';
+
+export const Router = express.Router();
+
+Router.get('/api/ping', async (req: Request, res: Response) => {
```

Comment:
I don't think routes should reside in the utils. Apps have their own endpoints we would have to respect and may not even use express.

Only the route handler logic should be here. In the current case it is just a `res.send()`. In the future it will be different stats and callbacks to add custom data to the response.